Dockerfile.12sp0

```
@@ -13,8 +17,9 @@ RUN zypper ar http://download.suse.de/ibs/SUSE:/SLE-12:/GA/standard/ SLE-12-stan
       vim osc ruby2.1-rubygem-gem2rpm hwinfo libx86emu1 zypper-migration-plugin sudo curl && \
     zypper --non-interactive rr SLE-12-standard SLE-12-update-standard SLE-12-ltss
 
-RUN wget http://username:password@gaffer.suse.de:9999/files/.regcode -O ~/.regcode
-RUN wget http://username:password@gaffer.suse.de:9999/files/.oscrc -O ~/.oscrc
+RUN printf "$REGCODE" >> ~/.regcode
+ADD integration/create-oscrc.sh /tmp/connect/integration/create-oscrc.sh
+RUN sh integration/create-oscrc.sh
```

this exec doesn't exist in the current working dir. Maybe you meant

```suggestion
RUN sh /tmp/connect/integration/create-oscrc.sh
```

:question:

---

thanks, changes applied!